federation/pkg/kubefed/init/init.go

```diff
@@ -109,8 +110,11 @@ func NewCmdInit(cmdOut io.Writer, config util.AdminConfig) *cobra.Command {
 		},
 	}
 
+	defaultImage := fmt.Sprintf("%s:%s", hyperkubeImageName, version.Get())
```

Comment:
@quinton-hoole Yeah, that's the most ideal default behavior. But there is no good/stable way to automatically detect the latest version right now. There is no API or anything as such to determine the latest version on the client machines and no technique is future proof. Given that, I think this is a reasonable default.